commands/sim.js

```diff
@@ -77,12 +80,6 @@ module.exports = function (program, conf) {
       so.selector = objectifySelector(selector || conf.selector)
       so.mode = 'sim'
 
-      if (cmd.conf) {
```

Comment:
I need to put this back in, actually, but in my opinion, above the bits for the CLI args so things passed in on the command line get the ultimate override. And/or we could move more things to stop using `options` and use the `conf` since it should contain a proper set of configuration, but that's going to be a change outside the scope for this PR.